import-service/serverless.ts

```diff
@@ -25,6 +25,7 @@ const serverlessConfiguration: Serverless = {
     },
     environment: {
       S3_BUCKET_NAME: 'rsapp-uploaded',
+      SQS_URL: 'https://sqs.eu-west-1.amazonaws.com/898792768163/catalogItemsQueue',
```

Comment:
I suggest you to add `Outputs` in `resource` section of your product-service `serverless.ts`.
```
Outputs: {
      SQSQueueUrl: {
        Value: { Ref: 'SQSQueue' },
      },
      SQSQueueArn: {
        Value: { 'Fn::GetAtt': [ 'SQSQueue', 'Arn' ] },
      },
}
```

Then you can use these values in your import-service `serverless.ts`.
`SQS_URL: '${cf:service-product-${self:provider.stage}.SQSQueueUrl}',`
And
```
{
    Effect: 'Allow',
    Action: 'sqs:*',
    Resource: '${cf:service-product-${self:provider.stage}.SQSQueueArn}',
}
```

Reply:
Thank you a lot for your comment
Due to lack of time will try to correct it during task 7